core/src/main/java/hudson/cli/DeleteViewCommand.java

```diff
@@ -87,16 +84,14 @@ protected int run() throws Exception {
                     throw e;
                 }
 
-                final String errorMsg = String.format(view_s + ": " + e.getMessage());
-                stderr.println(errorMsg);
+                stderr.println(view_s + ": " + e.getMessage());
                 errorOccurred = true;
-                continue;
```

Comment:
Why removing this?

Reply:
Useless :) it's the end of the loop, so it will continue anyway.

Reply:
Why is it the end? It's a for right?

Reply:
Oh I had my mind thinking it was a break! You are right!